### PR TITLE
BXMSPROD-1084: changed execution command

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_ 

[BXMSPROD-1084](https://issues.redhat.com/browse/BXMSPROD-1084)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* referenced pull request 1
* referenced pull request 2
* referenced pull request 2
etc. 

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: job-dsls/jobs/communityRelease_pipeline.groovy [hunk 1]
```diff
@@ -87,7 +87,7 @@ pipeline {
         // part of the Maven rep will be erased
         stage ('Remove M2') {
             steps {
-                sh "sh droolsjbpm-build-bootstrap/script/release/eraseM2.sh $m2Dir"
+                sh "./ droolsjbpm-build-bootstrap/script/release/eraseM2.sh $m2Dir"
```
Comment (Kaitou786) [inline job-dsls/jobs/communityRelease_pipeline.groovy:90]: For the executable to run, there should be no space between `./` and the file.

**Review by Kaitou786 — CHANGES_REQUESTED** [review]
Please see other occurrences

**Comment by mbiarnes** [comment]
replaced sh './droolsjbpm-build-bootstrap/... everywhere where before was used sh 'sh droolsjbpm-build-bootstrap/...

**Review by Kaitou786 — APPROVED** [review]
lgtm

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**